doc/source/examples/similarity_explanations_mnist.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<p>spelling</p><p> </p><p>actually, please rename to "Find similar instances"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>